art/metrics.py

```diff
@@ -3,8 +3,8 @@
 from art.utils.enums import TrainingStage
 
 if TYPE_CHECKING:
-    from art.core.base_components.base_model import ArtModule
-    from art.experiment.Experiment import ArtProject
+    from art.core import ArtModule
+    from art.project import ArtProject
```

Comment:
Weren't you going to remove the whole TYPE_CHECKING when types are put in quotes?

Reply:
Unfortunately this seems to be necessary for mypy